src/main/java/edu/harvard/iq/dataverse/MetadataBlockConverter.java

```diff
@@ -21,10 +21,10 @@
 public class MetadataBlockConverter implements Converter {
 
     //@EJB
-    DataverseServiceBean dataverseService = CDI.current().select(DataverseServiceBean.class).get();
+    MetadataBlockServiceBean dataverseService = CDI.current().select(MetadataBlockServiceBean.class).get();
 
     public Object getAsObject(FacesContext facesContext, UIComponent component, String submittedValue) {
-        MetadataBlock mdb = dataverseService.findMDB(new Long(submittedValue));
+        MetadataBlock mdb = dataverseService.findById(new Long(submittedValue));
```

Comment:
metadataBlockServiceBean.findById ?

Reply:
The `MetadataBlockServiceBean` already contained a method `findById()` with the same functionality. Therefore, I reused it there. https://github.com/IQSS/dataverse/blob/68e2136ca19996cade6a0031e4096ea1d6869cf9/src/main/java/edu/harvard/iq/dataverse/MetadataBlockServiceBean.java#L30